docs/ui/components/Link/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { css } from '@emotion/react';
+import { theme } from '@expo/styleguide';
+import NextLink from 'next/link';
+import React, { forwardRef } from 'react';
+
+export type LinkProps = React.PropsWithChildren<{
+  target?: string;
+  tabIndex?: number;
+  href?: string;
+  title?: string;
+  rel?: string;
+  as?: string;
+  isStyled?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  onMouseMove?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  onMouseEnter?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  onMouseLeave?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  onTouchStart?: (event: React.TouchEvent<HTMLAnchorElement>) => void;
+  testID?: string;
+  style?: React.CSSProperties;
+  className?: string;
+  openInNewTab?: boolean;
+  ariaLabel?: string;
+}>;
+
+export const Link = forwardRef<HTMLAnchorElement, LinkProps>(function Link(props, ref) {
```

Comment:
```suggestion
export const LinkBase = forwardRef<HTMLAnchorElement, LinkProps>(function Link(props, ref) {
```

Can we rename this to `LinkBase`? I want to do the same in the website to make it clear that `<A>` is preferred.

Reply:
Moved the Link component inside **ui/components/Text** and only export `A` tag. Kept the LinkBase name.